api/v1alpha7/types.go

```diff
@@ -163,6 +163,15 @@ type RootVolume struct {
 	AvailabilityZone string `json:"availabilityZone,omitempty"`
 }
 
+type DataVolume struct {
+	NameSuffix       string `json:"nameSuffix"`
+	Size             int    `json:"diskSize,omitempty"`
```

Comment:
`diskSize` for consistency with `rootVolume`, right? Do we want to change it?

Reply:
Similar to AZ, I'd rather match rootVolume and change them both at once in a future patch.